var/spack/repos/builtin/packages/py-py2cairo/package.py

```diff
@@ -35,7 +35,7 @@ class PyPy2cairo(Package):
 
     extends('python')
 
-    depends_on('cairo+X')
+    depends_on('cairo')
```

Comment:
So... one person says that `cairo+X` is required to build, another says that the `+X` breaks things.  Can anyone explain at a deeper level what the core problem is?